environment.yml

```diff
@@ -1,13 +1,18 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-methylseq-1.4
+name: nf-core-methylseq-1.4.1dev
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
+  # nf-core dependencies
+  - conda-forge::python=3.7.3
+  - conda-forge::markdown=3.1.1
+  - conda-forge::pymdown-extensions=6.0
+  - conda-forge::pygments=2.5.2
   - bioconda::fastqc=0.11.8
-  - conda-forge::r-markdown=1.1
+  - bioconda::multiqc=1.7
```

Comment:
Something isnt right in this file. `multiqc` is present twice and you dont seem to have all of the packages in the current `dev` branch?
https://github.com/nf-core/methylseq/blob/dev/environment.yml